
+++ //depot/CommEngine/patches/WLAN/PATCH_502L07_P1/data-model/cms-dm-tr181-wifi-eap.xml
--- //depot/CommEngine/patches/WLAN/PATCH_502L07_P1/data-model/cms-dm-tr181-wifi-eap.xml

@@ -1018,7 +1018,7 @@
 
   <parameter name="WlAuthMode" type="string" specSource="Broadcom" profile="Unspecified" supportLevel="ReadWrite" defaultValue="open" maxLength="18" />
 
-  <parameter name="WlAuthAkm" type="string" specSource="Broadcom" profile="Unspecified" supportLevel="ReadWrite" defaultValue="open" maxLength="18" validValuesArray="WlAuthAkmValues"/>
+  <parameter name="WlAuthAkm" type="string" specSource="Broadcom" profile="Unspecified" supportLevel="ReadWrite" defaultValue="open" maxLength="64" />
 
   <parameter name="X_BROADCOM_COM_WlAuth" type="int" specSource="Broadcom" profile="Unspecified" supportLevel="ReadWrite" defaultValue="0" />
 
@@ -1472,19 +1472,12 @@
   <element>WPA-Personal</element>
   <element>WPA2-Personal</element>
   <element>WPA-WPA2-Personal</element>
+  <element>WPA3-Personal</element>
+  <element>WPA3-Personal-Transition</element>
   <element>WPA-Enterprise</element>
   <element>WPA2-Enterprise</element>
   <element>WPA-WPA2-Enterprise</element>
 
-<validstringarray name="WlAuthAkmValues" />
-  <element>open</element>
-  <element>psk</element>
-  <element>psk2</element>
-  <element>psk psk2</element>
-  <element>wpa</element>
-  <element>wpa2</element>
-  <element>wpa wpa2</element>
-
 <validstringarray name="ifOperStatusValues" />
   <element>Up</element>
   <element>Down</element>
